.github/workflows/publish_plugin_to_hub.yml

```diff
@@ -104,9 +107,9 @@ jobs:
       - name: Run package command
         working-directory: ${{ needs.prepare.outputs.plugin_dir }}
         run: |
-          mkdir -p stubdocs
-          touch stubdocs/README.md
-          go run main.go package --docs-dir stubdocs -m ${{ steps.release-notes.outputs.result }} ${{ needs.prepare.outputs.plugin_kind }} ${{ needs.prepare.outputs.plugin_version }} .
+          rsync -v --exclude='tables.md' ../../../website/pages/docs/plugins/${{ needs.prepare.outputs.plugin_kind_plural }}/${{ needs.prepare.outputs.plugin_name }}/*.md docsdir/
```

Comment:
Needs https://github.com/cloudquery/cloudquery/pull/14498 to work. We exclude the tables since those are handled differently